docs/userGuide/contentAuthoring.md

```diff
@@ -287,11 +287,16 @@ This will result in `index.md` having the title `Binary Search Tree` and the spe
 ### Using Footers
 
 It is optional to put footers in your site. If you wish to do so, there are two ways that MarkBind supports.
+  
+1. [Specifying footer file in Front Matter](#specifying-footer-file-in-front-matter)
+2. [Defining your own inline footer](#defining-your-own-inline-footer)
 
-#### 1. Specifying footer file in Front Matter
+#### Specifying footer file in Front Matter
 
-By default, there is a `footer.md` file created for you in `_markbind/footers`. 
-If you need additional footers for different pages, you can create footer files in the same folder and wrap the contents in a `<footer>` tag. 
+Create a footer file in `_markbind/footers` and wrap the contents in a `<footer>` tag.
+You can create more than one footer file if you want different footers for different pages.
+
+A `footer.md` file has been created for you as a template in `_markbind/footers`, if you ran `init`.

```

Comment:
Tip: All new sites created with `init` will have a `footer.md` automatically created for you in `_markbind/footers`.